lib/android.dart

```diff
@@ -271,6 +274,8 @@ void _applyStylesXml({
   String? android12BackgroundColor,
   String? android12ImagePath,
   String? android12IconBackgroundColor,
+  String? android12BrandingImagePath,
+  String? android12BrandingImageDarkPath,
```

Comment:
`_applyStylesXml` is called separately for light and dark modes, so instead of including a `android12BrandingImageDarkPath` parameter, you should instead pass a dark value to `android12BrandingImagePath` on line 164.

Reply:
Yes you are right, I totally missed that one.
Updated the code with the latest commit.

Thanks for the heads up! :)